atmel-samd/Makefile

```diff
@@ -146,6 +146,10 @@ endif
 CFLAGS += -Os -DNDEBUG -flto $(GCC_INLINE_LIMIT)
 endif
 
+# Do extra optimization for speed on a few files.
+# e.g.:
+#$(BUILD)/xxx/Something.o: CFLAGS += $(CSUPEROPT)
```

Comment:
This isn't needed for this PR.

Reply:
I was basically just documenting how to do this, but I'll take it out.